Build/Scripts/runTests.sh

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env bash
+
+#
+# TYPO3 core test runner based on docker and docker-compose.
+#
+
+# Function to write a .env file in Build/testing-docker
+# This is read by docker-compose and vars defined here are
+# used in Build/testing-docker/docker-compose.yml
+setUpDockerComposeDotEnv() {
+    # Delete possibly existing local .env file if exists
+    [ -e .env ] && rm .env
+    # Set up a new .env file for docker-compose
+    {
+        echo "COMPOSE_PROJECT_NAME=local"
```

Comment:
Reusing `local` here may conflict with other runTests.sh based
projects. This should be unique.

Either hardcode here some project name (a valid one) or use
something more dynamiclly.

**NOTE:** That fact has come into play recently, specially
with raising `docker compose v2` usages. Therefore, not yet
mirrored in all runTests.sh implementations or the core itself.

Will provide them the next time.

For example, see:

https://github.com/web-vision/wv_deepltranslate/blob/9c0d396736f9e1f4d9a168d1105f03a174ca0c11/Build/Scripts/runTests.sh#L174-L181

Reply:
done.

Reply:
Wonderful, thank you!